gitpod-io/gitpod
# [ws-proxy] Pass invalid cookies to ports

This PR passes more cookies to services listing on workspace ports, even if those cookies have invalid names.

fixes https://github.com/gitpod-io/gitpod/issues/2470

### How to test
1. Save index.php with:
    ```PHP
    <?php
    setcookie("cookiename[0]", "something from somewhere");
    print_r($_COOKIE["cookiename"]);
    ?>
    ```
2. Serve with `php -S 0.0.0.0:8080`
3. Visit `$(gp url 8080)` and observe that the cookie value is printed. Please reload at least once s.t. the browser actually sets the cookie
